content/v2.0/influxdb-templates/stacks/_index.md

```diff
@@ -13,38 +11,104 @@ related:
   - /v2.0/reference/cli/influx/pkg/stack/
 ---
 
-**InfluxDB stacks** are **stateful [InfluxDB templates](/v2.0/influxdb-templates)**
-that let you add, update, and remove templated resources over time, avoid
-duplicating resources when applying the same or similar templates more than once,
-and apply changes to distributed instances of InfluxDB OSS or InfluxDB Cloud.
+**InfluxDB stacks** are stateful [templates](/v2.0/influxdb-templates) that let you apply changes
+to distributed instances of InfluxDB OSS or InfluxDB Cloud. Stacks (like templates) are groups of
+pre-defined resources, including buckets, dashboards, tasks, checks, labels, and Telegraf configurations.
 
-## Ideal use cases for InfluxDB stacks
-Stacks help save time and effort in the following use cases:
+- [Review ideal use cases for InfluxDB stacks](#review-ideal-use-cases-for-influxdb-stacks),
+  like automating InfluxDB deployments with GitOps and stacks.
+- [Manage InfluxDB stacks](#manage-influxdb-stacks)
 
-- [Actively develop and extend templates](#actively-develop-and-extend-templates)
+{{% note %}}
+**Key differences between stacks and templates**:
+
+- A template is a file outside of InfluxDB. Once a template is applied to InfluxDB, it becomes a stack.
+- Stacks only add a resource to an instance if the resource doesn't exist.
+Templates add all resources even if a resource already exists, which creates duplicate resources.
+  {{% /note %}}
+
+## Review ideal use cases for InfluxDB stacks
+
+Use stacks to save time in the following cases:
+
+- [Automate deployments with GitOps and stacks](#automate-deployments-with-gitops-and-stacks)
 - [Apply updates from source-controlled templates](#apply-updates-from-source-controlled-templates)
 - [Apply template updates across multiple InfluxDB instances](#apply-template-updates-across-multiple-influxdb-instances)
+- [Develop templates](#develop-templates)
 
-### Actively develop and extend templates
-InfluxDB stacks aid in developing and maintaining InfluxDB templates.
-Stacks let you modify and update template manifests and apply those changes in
-any stack that uses the template.
+### Automate deployments with GitOps and stacks
+
+GitOps is popular way to configure and automate deployments. Use InfluxDB stacks in a GitOps workflow
+to automatically update distributed instances of InfluxDB OSS or InfluxDB Cloud.
+
+#### Set up a GitHub repository to back your InfluxDB instance
+
+Use our sample [`influxDB-assets` repository](https://github.com/russorat/influxdb-assets) to get started.  
```

Comment:
should we move this to the influxdata org?

Reply:
Excellent suggestion :)